IntegrationTestDS3/IntegrationTestDS3Client.cs

```diff
@@ -326,6 +326,18 @@ public void TestSpecialCharacterInObjectName()
                     select f;
 
                 Assert.AreEqual(1, putfile.Count());
+
+                // Does a query param escape properly?
+                GetObjectsRequest getObjectsWithNameRequest = new GetObjectsRequest();
+                getObjectsWithNameRequest.ObjectName = fileName;
+                var getObjectsResponse = _client.GetObjects(getObjectsWithNameRequest);
+
+                var filename =
+                    from f in getObjectsResponse.Objects
+                    where f.Name == fileName
+                    select f;
+
+                Assert.AreEqual(1, filename.Count());
```

Comment:
ok, got it.  I thought it through a bit more and realized that its getting "un-escaped" on the way back.